src/views/tables/OrdersTable/OrderActionsCell.tsx

```diff
@@ -48,22 +53,32 @@ export const OrderActionsCell = ({ orderId, orderFlags, status, isDisabled }: El
 
   return (
     <ActionsTableCell>
-      <$CancelButton
-        key="cancelorder"
-        iconName={IconName.Close}
-        iconSize="0.875em"
-        shape={ButtonShape.Square}
-        $uiRefreshEnabled={uiRefresh}
-        {...(isOrderStatusClearable(status)
-          ? { onClick: () => dispatch(clearOrder(orderId)) }
-          : {
-              onClick: onCancel,
-              state: {
-                isLoading: isCanceling,
-                isDisabled: isCancelDisabled,
-              },
-            })}
-      />
+      <WithTooltip
+        side="left"
+        tw="[--tooltip-backgroundColor:--color-layer-5]"
+        tooltipString={
+          isOrderStatusClearable(status)
+            ? stringGetter({ key: STRING_KEYS.CLEAR })
+            : stringGetter({ key: STRING_KEYS.CANCEL_ORDER })
+        }
+      >
+        <$CancelButton
+          key="cancelorder"
+          iconName={IconName.Close}
+          iconSize="0.875em"
+          shape={ButtonShape.Square}
+          $uiRefreshEnabled={uiRefresh}
+          {...(isOrderStatusClearable(status)
+            ? { onClick: () => dispatch(clearOrder(orderId)) }
+            : {
+                onClick: onCancel,
+                state: {
+                  isLoading: isCanceling,
+                  isDisabled: isCancelDisabled,
+                },
+              })}
+        />
+      </WithTooltip>
```

Comment:
nit: can we make the loading state align right?
<img width="279" alt="image" src="https://github.com/user-attachments/assets/52b3cebb-96c1-47b0-8575-36b19d6d9d23">

Reply:
oops yes will dooo